.ci/generate-website.sh

```diff
@@ -16,6 +16,7 @@ echo TARGET_VERSION="$TARGET_VERSION"
 git checkout "checkstyle-$TARGET_VERSION"
 
 echo "Generating web site"
-./mvnw -e --no-transfer-progress site -Pno-validations -Dmaven.javadoc.skip=false
+./mvnw -e --no-transfer-progress site -Pno-validations \
+  -Dmaven.javadoc.skip=false -Djdepend.skip=false
```

Comment:
Good update, this script is used in release https://github.com/checkstyle/checkstyle/blob/c385e4ea8b921b32f515a85379128df8716e8efd/.github/workflows/release-update-github-io.yml#L43